test/prepare.sh

```diff
@@ -2,18 +2,39 @@
 
 set -eux -o pipefail
 
-IMG="${IMG:-controller:test}"
+IMG="${IMG:-localhost/controller:test}"
 LOGDIR="${LOGDIR:-/tmp/logs}"
+CONTAINER_RUNTIME="${CONTAINER_RUNTIME:-}"
 
 mkdir -p "${LOGDIR}"
 
+if [ -z "${CONTAINER_RUNTIME}" ]; then
```

Comment:
```suggestion
if [[ -z "${CONTAINER_RUNTIME}" ]]; then
```
Since we're doing bash, let's use bash test operator.